src/main/drivers/serial_usb_vcp.c

```diff
@@ -193,7 +196,11 @@ serialPort_t *usbVcpOpen(void)
 
     IOInit(IOGetByTag(IO_TAG(PA11)), OWNER_USB, 0);
     IOInit(IOGetByTag(IO_TAG(PA12)), OWNER_USB, 0);
+#ifdef USB_CDC_HID
+    USBD_Init(&USB_OTG_dev, USB_OTG_FS_CORE_ID, &USR_desc, &USBD_HID_CDC_cb, &USR_cb);
```

Comment:
Is this OK? Composite device is initialized on serial open ... 
At least some comment should explain the situation

Reply:
Well it happens on initializing other serial ports. Technically this initializes CDC USB Device without composite option, not just opening VCP.